### ISSUE TITLE
Error running "make targets"

### ISSUE THREAD

**Issue by dwang159** [linkedin/iris#473]
I am getting below error while running make targets on iris.

File "/iris/src/iris/bin/sync_targets.py", line 145, in <dictcomp>
    if user['active']}
TypeError: string indices must be integers
Makefile:13: recipe for target 'targets' failed
make: *** [targets] Error 1

Below are the logs from oncall server.

[10/Jan/2019:04:24:05 +0000] "GET /api/v0/users?fields=name&fields=contacts&fields=active HTTP/1.1" 401 85 "-" "python-requests/2.20.0"

_Originally posted by @adityavijjanT in https://github.com/linkedin/iris/pull/333#issuecomment-452990413_

**Comment by adityavijjanT** [comment]
Thanks for the quick reply, I have used default config file from both oncall and iris repo, have only changed db information nd ldap integration oncall config files. 

oncall config:-

 - type: dummy
    application: oncall
    iris_api_key: magic

############################
### Oncall frontend settings
############################
supported_timezones:
  - 'US/Pacific'
  - 'US/Eastern'
  - 'US/Central'
  - 'US/Mountain'
  - 'US/Alaska'
  - 'US/Hawaii'
  - 'Asia/Kolkata'
  - 'Asia/Shanghai'
  - 'UTC'

index_content_setting:
  # Page footer contents
  footer: |
    <ul>
      <li>Oncall © LinkedIn 2017</li>
      <li>Feedback</li>
      <li><a href="http://oncall.tools" target="_blank">About</a></li>
    </ul>
  # Note displayed when a user has no phone number
  missing_number_note: 'No number'
header_color: '#3a3a3a'

# Integration with Iris, allowing for escalation from Oncall
iris_plan_integration:
  activated: True
  # Iris app and key settings
  app: oncall
  api_key: magic
  api_host: http://iris:16649


Iris Config
## Enabled modules for resolving complex user roles to individual usernames, processed
## in the order listed here.
role_lookups:
  - user # support for 'role' being username
  - mailing_list # mailing list config; see bottom of file
  - oncall # oncall service integration
  - dummy # dummy role lookup used for testing purposes. Disable this in production.

oncall-api: http://oncall:8080  # oncall url
oncall-app: oncall # application for oncall auth
oncall-key: magic # API key for oncall auth

applications:
 - dummy_app

**Comment by alexw19** [comment]
Curious if this issue was resolved as I'm having the same problem.

**Comment by adityavijjanT** [comment]
yes this is resolved. i have sync users from oncall instead of ldap.

ldap --> oncall --> iris

https://iris.claims/docs/installation.html#integrating-iris-and-oncall

make sure your oncall api key and application are correctly configured in iris config file.

**Comment by TomMir** [comment]
After you have edited the config, what command do you run and from where?
